jina/helper.py

```diff
@@ -597,13 +597,34 @@ def get_full_version() -> str:
 
 
 def is_url(text):
+    ul = '\u00a1-\uffff'  # Unicode letters range (must not be a raw string).
```

Comment:
@nan-wang , shouldn't we have the `url_pat` regex compiled statically? outside this function? This compilation could slow a little bit the `request` generation right?

Reply:
yes, `url_pat` should be moved outside